apps/gallery/js/thumbnail_date_group.js

```diff
@@ -139,9 +134,11 @@ ThumbnailDateGroup.formatter = new navigator.mozL10n.DateTimeFormat();
 
 ThumbnailDateGroup.prototype.localize = function() {
   var date = new Date(this.date);
-  var format = navigator.mozL10n.get('date-group-header');
-  var formattedDate = ThumbnailDateGroup.formatter.localeFormat(date, format);
-  this.header.textContent = formattedDate;
+  navigator.mozL10n.formatValue('date-group-header', function(format) {
+    var formattedDate =
+      ThumbnailDateGroup.formatter.localeFormat(date, format);
+    this.header.textContent = formattedDate;
```

Comment:
agh, I forgot that we switched formatValue to be a Promise. Fixed in the updated patch :)